test/regressions/site/src/tests/Input/Inputs.js

```diff
@@ -30,29 +35,35 @@ export default class Inputs extends Component {
     const classes = this.context.styleManager.render(styleSheet);
 
     return (
-      <div className={classes.container}>
-        <Input
-          defaultValue="Hello world"
-          className={classes.input}
-        />
-        <Input
-          placeholder="Placeholder"
-          className={classes.input}
-        />
-        <Input
-          value="Disabled"
-          className={classes.input}
-          disabled
-        />
-        <Input
-          error
-          value="Error"
-          className={classes.input}
-        />
+      <div>
+        <div className={classes.container}>
+          <Input
+            value="Hello world"
+            className={classes.input}
+          />
+          <Input
+            placeholder="Placeholder"
+            className={classes.input}
+          />
+          <Input
+            value="Disabled"
+            className={classes.input}
+            disabled
+          />
+          <Input
+            error
+            value="Error"
+            className={classes.input}
+          />
+          <Input
+            value="Focused"
+            ref={(c) => { this.focusInput = c; }}
+            className={classes.input}
+          />
+        </div>
         <Input
-          value="Focused"
-          ref={(c) => { this.focusInput = c; }}
-          className={classes.input}
+          value="Large input"
+          className={classNames(classes.input, classes.large)}
```

Comment:
ah, I must have something different in my app, you are right with your original code.